apps/frontend-manage/.env.prd

```
@@ -0,0 +1,24 @@
+COOKIE_DOMAIN=.klicker.prd.df-app.ch
+API_DOMAIN=backend-sls$COOKIE_DOMAIN
+
+NEXT_PUBLIC_AVATAR_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/avatars"
+NEXT_PUBLIC_API_URL="https://$API_DOMAIN/api/graphql"
+NEXT_PUBLIC_API_URL_SSR="http://klicker-uzh-qa-klicker-uzh-v3-backend-graphql:3000/api/graphql"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)